databricks/sdk/core.py

```diff
@@ -12,8 +10,8 @@
 from .config import *
 # To preserve backwards compatibility (as these definitions were previously in this module)
 from .credentials_provider import *
-from .errors import DatabricksError, error_mapper
-from .errors.private_link import _is_private_link_redirect
+from .errors import DatabricksError, _get_api_error
```

Comment:
Will do.

What do you think is a good way to indicate that the function is internal to the SDK but not to the `errors` module?

Reply:
To be honest, I don't have a definite answer here. Without changing the code structure, I would simply make that explicit in the function's doc. Otherwise, I would try to organize the `core` package as a single "umbrella" interface to internal modules that start with a `_` (e.g. `core/_errors.py`). I've also seen people explicitly put private modules in a `private` directory (e.g. `core/private/errors.py`). I'll do some research to see if this is covered by our internal style guide.